cloudsync/event.py

```diff
@@ -286,17 +302,15 @@ def _fill_event_path(self, event: Event):
         state = self.state.lookup_oid(self.side, event.oid)
         if state:
             event.path = state[self.side].path
-        if not event.path and event.exists:
-            info = self.provider.info_oid(event.oid)
-            if info:
-                event.path = info.path
-                event.otype = info.otype
-                event.hash = info.hash
+        if not event.path and event.exists is True:
```

Comment:
Do we want to do this if `event.exists` is None?

Reply:
yeah i think so